### PR TITLE
Additional elements and changes

### PR DESCRIPTION
**Pull request by vk2him**
Please review and advise

### CODE REVIEW

File: custom_components/clientraw/sensor.py [hunk 1]
```diff
@@ -295,16 +327,16 @@ def try_again(err: str):
...
-                       "sleetshowers", "snow", "snowmelt", "snowshowers2",
-                       "sunny", "thundershowers", "thundershowers2",
-                       "thunderstorms", "tornado", "windy", "stopped",
-                       "rainning", "wind + rain"]
+                       "night snow", "night thunder", "overcast",
+                       "partly cloudy", "rain", "rain2", "showers", "sleet",
+                       "sleet showers", "snow", "snow melt", "snow showers2",
+                       "sunny", "thunder showers", "thunder showers2",
+                       "thunder storms", "tornado", "windy",
+                       "stopped raining"]
```
Comment (pilotak) [inline custom_components/clientraw/sensor.py]: Well spotted  `stoped raining`
I would like to keep the `wind + rain` there is apparently also `sunrise` and `sunset`
https://git.dev.hsdn.org/pub/wdparser/src/master/src/WeatherDisplay.php#L1203

File: custom_components/clientraw/sensor.py [hunk 2]
```diff
@@ -226,7 +255,7 @@ def try_again(err: str):
                 new_state = round(speed, 2)
 
             elif dev.type == 'wind_gust':
-                gust = float(self.data[2])
+                gust = float(self.data[133])
```
Comment (pilotak) [inline custom_components/clientraw/sensor.py:259]: Why not use index 71?

**Comment by pilotak** [comment]
Good job in finding those bugs. Could you please alter the file `info.md` - this is the one showed in HA: it's basically the same as README

File: custom_components/clientraw/sensor.py (hunk 2 above)
Comment (vk2him) [reply custom_components/clientraw/sensor.py:259]: I think 71 is max gust for the day? Usually a weather briefing will talk about current winds and current gusts - I might change the name to be wind_gust_hour and add 71 as wind_gust_day ...?

File: custom_components/clientraw/sensor.py (hunk 1 above)
Comment (vk2him) [reply custom_components/clientraw/sensor.py]: Sure - I hadn't realised that - the parser I use doesn't have those extras so I will add

**Comment by vk2him** [comment]
I'll make the changes tomorrow thanks

File: custom_components/clientraw/sensor.py (hunk 1 above)
Comment (vk2him) [reply custom_components/clientraw/sensor.py]: Sorry for delay - ok those added

**Comment by vk2him** [comment]
Please let me know about the changes thanks

File: README.md [hunk 3]
```diff
@@ -72,47 +79,53 @@ sensor:
...
-22 = showers2
+22 = showers
 23 = sleet
-24 = sleetshowers
+24 = sleet showers
 25 = snow
-26 = snowmelt
-27 = snowshowers2
+26 = snow melt
+27 = snows howers2
```
Comment (pilotak) [inline README.md]: Typo here

File: custom_components/clientraw/sensor.py (hunk 2 above)
Comment (pilotak) [reply custom_components/clientraw/sensor.py:259]: Please do so

File: info.md [hunk 4]
```diff
@@ -20,20 +20,44 @@ Configuration variables:
...
+  - **yearly_rain**: Daily Rain (mm or in)
+  - **rain_rate**: Daily Rain (mm or in)
   - **pressure**: Pressure (hPa or inHg)
   - **humidity**: Relative humidity (%)
+  - **humidity_indoor**: Relative humidity (%)
   - **cloud_height**: Cloud Height (m or ft)
   - **forecast**: string based output ie.: night showers
 
-A full configuration example can be found below:
+## Install via [HACS](https://github.com/custom-components/hacs)
```
Comment (pilotak) [inline info.md]: there is no need to have an install info in HACS store, there is already a button to install

**Review by pilotak — CHANGES_REQUESTED** [review]

**Review by pilotak — APPROVED** [review]

**Comment by pilotak** [comment]
@vk2him  sorry i approved too early could you please delete install part from `info.md` and `wind_gust` rename to per hour

**Comment by vk2him** [comment]
Yes no problem, will do shortly thanks

File: custom_components/clientraw/sensor.py (hunk 2 above)
Comment (vk2him) [reply custom_components/clientraw/sensor.py:259]: Ok these now added

**Comment by vk2him** [comment]
Please review, you might want to bump the version number though. Thanks

**Comment by pilotak** [comment]
Thanks, almost perfect  - could you please delete the installation tutorial in `info.md `?

**Comment by vk2him** [comment]
Ok, info.md changed. Thanks

**Comment by pilotak** [comment]
Thank you

**Pull request merged**